unicorecmsffl/templates/base.pt

```diff
@@ -85,6 +85,8 @@
 
     </div>
 
+    <div metal:define-slot="search" metal:use-macro="view.search_box_template"></div>
```

Comment:
Is there a way we can pull `search_box_template` template in without adding an extra `div`?